core/templates/dev/head/editor/StateResponses.js

```diff
@@ -306,6 +306,38 @@ oppia.controller('StateResponses', [
       trainingDataService.initializeTrainingData(
         explorationId, currentStateName);
     };
+    $scope.suppressDefaultAnswerGroupWarnings = function() {
+      var interactionId = $scope.getCurrentInteractionId();
+      if (interactionId === 'MultipleChoiceInput') {
+        var choices = $scope.getAnswerChoices().length;
+        var suppressAnswer = responsesService.getAnswerGroups();
+        var defaultAnswers;
+        // For choice length, have indices 0 through choice length - 1
+        for (var x = 0; x < choices; x++) {
+          if (suppressAnswer['' + x + ''] === undefined) {
+            // Return false if no object exists for the index
+            /* Note that multiple choice only allows for
```

Comment:
Yep, but I think you should be able to subsequently add a new answer that coincides with a different choice? So if the multiple-choice options are A, B, C and D, then the first answer group could handle both options A and B, for example (both still using "Equal To").